.github/workflows/main.yml

```diff
@@ -108,7 +108,7 @@ jobs:
         files: |
           target/wasm32-unknown-unknown/release/wasi_snapshot_preview1.wasm
           wasi_snapshot_preview1.command.wasm
```

Comment:
Should we also rename `wasi_snapshot_preview1.command.wasm` to `.cli.wasm`?

Reply:
Yeah we should, though from talking to Luke I think I'm going to change renaming the world to `command` (and the `command` function to `main`), so I'll hold off on this for now.